src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -52,6 +59,13 @@
 
 		<Hr />
 
+		<ButtonMenu ariaLabel={$i18n.wallet_connect.text.name} on:click={openWalletConnectAuth}>
```

Comment:
Previously the button was only available when WalletConnect was not connected. What happens here if the user click on this button once the connection is established?

Reply:
wait, sorry, i forgot to push the commit where we disable the button once connected (as per requirements)

check last commit